package.json

```diff
@@ -1,8 +1,14 @@
 {
-  "name": "snjs",
-  "version": "2.0.3",
+  "name": "@standardnotes/snjs",
+  "version": "2.0.4",
+  "engines": {
```

Comment:
What does the engines key do?

Reply:
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines ;)

Reply:
Hah, fair enough, sorry~ in that case I assume this is strictly used to help one build with a correct version of node, in which case maybe `>=12.19.0 <14.x` would likely be more correct? I'm running node 14.14.0 and building without issue for example